hypha/cookieconsent/templates/includes/banner.html

```diff
@@ -2,14 +2,52 @@
 
 {% if show_banner %}
     <div class="wrapper wrapper--cookieconsent cookieconsent">
-        <div class="wrapper wrapper--small">
-            <h2 class="cookieconsent__title">{% trans title %}</h2>
-            <div class="cookieconsent__message rich-text">
-                {{ message|richtext }}
+        <div class="cookieconsent__content">
+            <div class="cookieconsent__brief">
+                <div>
+                    <h2 class="cookieconsent__title">{% trans settings.cookieconsent_title %}</h2>
+                    <div class="cookieconsent__message rich-text">
+                        {{ settings.cookieconsent_message|richtext }}
+                    </div>
+                    <div class="cookieconsent__actions">
+                        {% if settings.cookieconsent_analytics %}
+                            <button class="button button--cookieconsent button--decline" title="{% trans 'Decline tracking cookies.' %}" type="button" data-consent="decline">{% trans 'Essential only' %}</button>
+                            <button class="button button--cookieconsent button--accept lg:ms-[20px]" title="{% trans 'Accept tracking cookies.' %}" type="button" data-consent="accept">{% trans 'Accept all' %}</button>
+                        {% else %}
+                            <button class="button button--cookieconsent button--accept" title="{% trans 'Acknowledge use of essential cookies.' %}" data-consent="ack">{% trans 'Ok' %}</button>
+                        {% endif %}
+                        <button class="button button--cookieconsent button--learn-more lg:ms-[20px]" title="{% trans 'Learn more about specific cookies used.' %}" show-learn-more="true">{% trans 'Learn More' %}</button>
+                    </div>
+                </div>
             </div>
-            <div class="cookieconsent__actions">
-                <button class="button button--cookieconsent button--decline" title="{% trans 'Decline tracking cookies.' %}" type="button" data-consent="false">{% trans 'Decline' %}</button>
-                <button class="button button--cookieconsent button--accept button--left-space" title="{% trans 'Accept tracking cookies.' %}" type="button" data-consent="true">{% trans 'Accept' %}</button>
+            <div class="cookieconsent__learnmore">
+                <div>
+                    <div class="cookieconsent__statement">
+                        <p>This Website uses cookies.  Below you can select the categories of cookies to allow when you use our website and services.  For more information, please see our Privacy Policy.</p>
+                    </div>
+                    <div class="cookieconsent__info-wrapper">
+                        <div class="cookieconsent__info">
+                            <p class="font-semibold">{% trans 'Essential Cookies' %}</p>
+                            {{ settings.cookieconsent_essential_about|richtext }}
+                            <span class="text-green-500">Always enabled.</span>
```

Comment:
This green has a low contrast against the blue background. Maybe make it bold white instead?

<img width="962" alt="Skärmavbild 2024-06-19 kl  08 16 25" src="https://github.com/HyphaApp/hypha/assets/191392/73053738-1354-4999-83ab-5b0a2227000d">